Cargo.toml

```diff
@@ -49,3 +49,135 @@ default = ["chrono", "unix-socket", "tls"]
 unix-socket = ["hyperlocal"]
 tls = ["openssl", "hyper-openssl"]
 vendored-ssl = ["tls", "openssl/vendored"]
+
+[[example]]
```

Comment:
Thanks for the contribution.

I don't believe these are actually needed unless we were to name one of the examples something other than the name of the file.

Reply:
Cargo uses the convention of resolving an example by file name by default

Reply:
I didn't know! 
I close the PR
Thanks